### PR TITLE
source-han-*: install instead of ln

### PR DESCRIPTION
**Pull request by linsui**
###### Motivation for this change
The LibreOffice can't recognize the symlink for some unknown reasons so I revert https://github.com/NixOS/nixpkgs/pull/95701

CC @taku0 @emilazy @gebner

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please use only fetchzip and postFetch to install the font files.

**Comment by linsui** [comment]
Done. Is this ok?

File: pkgs/data/fonts/source-han/default.nix [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-    Family =
+  makePackage = {
+    family,
+    description,
+    rev,
+    sha256,
+    postFetch ? ''
+      install -m444 -Dt $out/share/fonts/opentype/source-han-${family} $downloadedFile
+    '',
+    zip ? ""
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/source-han/default.nix:15]:
```suggestion
  makePackage = { 
    family
    , description
    , rev
    , sha256
    ,  postFetch ? ''
        install -m444 -Dt $out/share/fonts/opentype/source-han-${family} $downloadedFile
      ''
    , zip ? ""
```
or just use nixpkgs-fmt which probably has a better idea what to do.

Comment (linsui) [reply pkgs/data/fonts/source-han/default.nix:15]: Thanks! I tested with nixfmt but the result looks worse. nixpkgs -fmt works better for me.

**Pull request merged**